### PR TITLE
fix(show-monitor): support multi-tenant cluster

### PR DESCRIPTION
**Pull request by juliayakovlev**
Restore monitor for multi-tenant cluster.
Now Nemesis dashboard of every cluster is point to the last cluster.

Task: https://github.com/scylladb/scylla-cluster-tests/issues/5458

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@juliayakovlev you test it's working also for a regular jobs ? (i.e. not k8s or multi-tenant) ?

**Review by fruch — APPROVED** [review]
LGTM

**Comment by juliayakovlev** [comment]
@fruch 
I tested it for single cluster job and this that in the task.

File: sdcm/monitorstack/__init__.py [hunk 1]
```diff
@@ -299,17 +329,13 @@ def run_monitoring_stack_containers(monitoring_stack_dir, monitoring_data_dir, s
...
-        LOGGER.warning('There is no dashboard %s. defaults to master dashboard', sct_dashboard_file_name)
+def restore_sct_dashboards(grafana_docker_port, sct_dashboard_file):
+    if not sct_dashboard_file:
+        LOGGER.warning('There is no dashboard %s. defaults to master dashboard', sct_dashboard_file)
         sct_dashboard_file_name = "scylla-dash-per-server-nemesis.master.json"
-        sct_dashboard_files = [Path(__file__).parent.parent.parent / 'data_dir' / sct_dashboard_file_name]
+        sct_dashboard_file = [Path(__file__).parent.parent.parent / 'data_dir' / sct_dashboard_file_name]
 
     dashboard_url = f'http://localhost:{grafana_docker_port}/api/dashboards/db'
-    sct_dashboard_file = sct_dashboard_files[0]
```
Comment (vponomaryov) [inline sdcm/monitorstack/__init__.py:312]: I think removal of that line must break the code. On the following line with `with` statement we expect there a string-like object not list.

Have you tested it?

Comment (fruch) [reply sdcm/monitorstack/__init__.py:312]: to get to a point you don't have a dashboard, is was back when we used the per version dashboard, and it was missing.
it's rarely would happen now.

but yes this fallback won't work...

**Pull request merged**